apps/user/serializers.py

```diff
@@ -166,6 +166,28 @@ def validate(self, attrs):
         raise serializers.ValidationError(gettext('Activation link is not valid.'))
 
 
+class VerifySerializer(serializers.Serializer):
```

Comment:
Let's Use ModelSerializer, and use fields = ('is_verified',)
```suggestion
class UserVerifySerializer(serializers.ModelSerializer):
```